ddl/db_partition_test.go

```diff
@@ -117,7 +118,7 @@ func (s *testIntegrationSuite) TestCreateTableWithPartition(c *C) {
 	_, err = tk.Exec(`CREATE TABLE rc (
     		a INT NOT NULL,
     		b INT NOT NULL,
-			c INT NOT NULL
```

Comment:
Why do this？

Reply:
Because it mix whitespace and tab here, and looks weird in my editor.